meta-ostro-xt/recipes-graphics/xorg-driver/xf86-video-intel_%.bbappend

```diff
@@ -0,0 +1,8 @@
+SRC_URI = "file://always_include_xorg_server.h.patch \
```

Comment:
The patches are in ostro-os/meta/recipes-graphics/xorg-driver/xf86-video-intel

The file ostro-os/meta/recipes-graphics/xorg-driver/xf86-video-intel_2.99.917.bb are including these plus the patch "file://disable-x11-dri3.patch". In order to get rid of this dri3 disable patch, the SRC_URI has been redefined in the bbappend file.

This builds fine on my machine.